json_rpc/clients/websocketclient.nim

```diff
@@ -42,18 +42,20 @@ method call*(self: RpcWebSocketClient, name: string,
 
 proc processData(client: RpcWebSocketClient) {.async.} =
   var error: ref CatchableError
+  let ws = client.transport
   try:
-    while true:
-      var value = await client.transport.receiveString()
-      if value == "":
+    while ws.readystate != ReadyState.Closed:
+      var value = await ws.recv()
+
+      if value.len == 0:
         # transmission ends
         break
 
-      client.processMessage(value)
+      client.processMessage(cast[string](value))
```

Comment:
use `string.fromBytes` from https://github.com/status-im/nim-stew/blob/master/stew/byteutils.nim for string conversions

Reply:
fixed